src/layout/section/landingGridSection/LandingGridSection.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+
+import { HeroSection, HeroSectionProps } from '../heroSection'
+import { LandingGridSectionElements } from './LandingGridSection.style'
+
+export type LandingGridSectionProps = HeroSectionProps &
+  Readonly<{
+    children: React.ReactNode
+  }>
+
+export const LandingGridSection = ({
+  children,
+  heroText,
+  heroImageUrlLarge,
+  heroImageUrlSmall,
+}: LandingGridSectionProps) => (
+  <LandingGridSectionElements.Grid>
+    <LandingGridSectionElements.Body>{children}</LandingGridSectionElements.Body>
+    <LandingGridSectionElements.Figure>
+      <HeroSection
```

Comment:
We should not have a section in a section.

Reply:
Why does?
https://html.spec.whatwg.org/multipage/sections.html#headings-and-sections

Anyway, I'm considering dropping `HeroSection` since it doesn't cover 100 of my case

Reply:
HeroSection was dropped